happybase/client.py

```diff
@@ -1,25 +1,40 @@
 # -*- coding: utf-8 -*-
+
+import time
+import logging
+import datetime
+from functools import partial
+from socket import error as socket_error
+from collections import deque, defaultdict
+
+import numpy as np
 from thriftpy.thrift import TClient
 from thriftpy.thrift import TApplicationException
 from thriftpy.transport import TTransportException
-from socket import error as socket_error
-from collections import deque
-from time import sleep, time
-import logging
-import numpy as np
+
 
 logger = logging.getLogger(__name__)
 
 
+TIMINGS_KEEP_LIMIT = 50
+EPOCH_DATETIME = datetime.datetime.utcfromtimestamp(0)
+
+
+def utc_timestamp():
+    return (datetime.datetime.utcnow() - EPOCH_DATETIME).total_seconds()
```

Comment:
Before this PR `time.time()` was used everywhere in the module, but as we need to send some timestamps for the metrics to create time-series, it's better/simpler to use UTC timestamps from the beginning.